clever/launch/main_camera.launch

```diff
@@ -1,20 +1,19 @@
 <launch>
-    <!-- Camera position and orientation are represented by base_link -> main_camera_optical transform -->
-    <!-- static_transform_publisher arguments: x y z yaw pitch roll frame_id child_frame_id -->
-
     <!-- article about camera setup: https://clever.coex.tech/camera_frame -->
 
-    <!-- camera is oriented downward, camera cable goes backward [option 1] -->
-    <node pkg="tf2_ros" type="static_transform_publisher" name="main_camera_frame" args="0.05 0 -0.07 -1.5707963 0 3.1415926 base_link main_camera_optical"/>
+    <arg name="orientation_z" default="down"/> <!-- direction the camera points: down, up -->
+    <arg name="orientation_y" default="backward"/> <!-- direction the camera cable points: backward, forward -->
 
-    <!-- camera is oriented downward, camera cable goes forward  [option 2] -->
-    <!--<node pkg="tf2_ros" type="static_transform_publisher" name="main_camera_frame" args="0.05 0 -0.07 1.5707963 0 3.1415926 base_link main_camera_optical"/>-->
+    <!-- Camera position and orientation are represented by base_link -> main_camera_optical transform -->
+    <!-- static_transform_publisher arguments: x y z yaw pitch roll frame_id child_frame_id -->
```

Comment:
Maybe this could be shifted down, closer to the "custom template" part?
`frame_id` and `child_frame_id`, while absolutely correct, are not very clear. Will this add a new transform from `frame_id` to `child_frame_id`? Or is it the other way round?

Reply:
Solved